app/person-escort-record/controllers.js

```diff
@@ -0,0 +1,30 @@
+const wizard = require('hmpo-form-wizard')
+
+const FormWizardController = require('../../common/controllers/form-wizard')
+
+function sectionHandler(req, res, next) {
+  const { questions } = req.framework
+  const { key: sectionKey, steps: sectionSteps } = req.frameworkSection
+  const wizardConfig = {
+    buttonText: 'actions::save_and_continue',
+    controller: FormWizardController,
+    entryPoint: true,
+    journeyName: `person-escort-record-${sectionKey}`,
+    journeyPageTitle: 'Person escort record',
+    name: `person-escort-record-${sectionKey}`,
+    template: 'form-wizard',
+  }
+  const steps = {
+    '/': {
+      reset: true,
+      resetJourney: true,
+    },
+    ...sectionSteps,
+  }
+
+  return wizard(steps, questions, wizardConfig)(req, res, next)
```

Comment:
If you passed the framework section as a string (as above) rather than the whole object, we could cache and reuse the middleware returned by the wizard.